bugwarrior/services/deck.py

```diff
@@ -0,0 +1,184 @@
+        return response.json()
+
+    # see https://deck.readthedocs.io/en/latest/API/#stacks for API docs
+    def get_stacks(self, board_id):
+        response = self.session.get(f'{self.base_uri}/{self.api_base_path}/boards/{board_id}/stacks')
+        return response.json()
+
+
+class NextcloudDeckIssue(Issue):
+    AUTHOR = 'nextclouddeckauthor'
+    BOARD_ID = 'nextclouddeckboardid'
+    BOARD_TITLE = 'nextclouddeckboardtitle'
+    STACK_ID = 'nextclouddeckstackid'
+    STACK_TITLE = 'nextclouddeckstacktitle'
+    CARD_ID = 'nextclouddeckcardid'
+    TITLE = 'nextclouddeckcardtitle'
+    DESCRIPTION = 'nextclouddeckdescription'
+    ORDER = 'nextclouddeckorder'
+    ASSIGNEE = 'nextclouddeckassignee'
+
+    UDAS = {
+        AUTHOR: {
+            'type': 'string',
+            'label': 'Nextcloud Deck Issue Author',
+        },
+        BOARD_ID: {
+            'type': 'numeric',
+            'label': 'Nextcloud Deck Board ID',
+        },
+        BOARD_TITLE: {
+            'type': 'string',
+            'label': 'Nextcloud Deck Board Title',
+        },
+        STACK_ID: {
+            'type': 'numeric',
+            'label': 'Nextcloud Deck Stack ID',
+        },
+        STACK_TITLE: {
+            'type': 'string',
+            'label': 'Nextcloud Deck Stack Title',
+        },
+        CARD_ID: {
+            'type': 'numeric',
+            'label': 'Nextcloud Deck Card ID',
+        },
+        TITLE: {
+            'type': 'string',
+            'label': 'Nextcloud Deck Card Title',
+        },
+        DESCRIPTION: {
+            'type': 'string',
+            'label': 'Nextcloud Deck Card Description',
+        },
+        ORDER: {
+            'type': 'numeric',
+            'label': 'Nextcloud Deck Order',
+        },
+        ASSIGNEE: {
+            'type': 'string',
+            'label': 'Nextcloud Deck Assignee(s)',
+        }
+    }
+
+    UNIQUE_KEY = (BOARD_ID, STACK_ID, CARD_ID,)
+
+    def to_taskwarrior(self):
+        return {
+            'project': self.extra['board']['title'].lower().replace(' ', '_'),
+            'priority': self.get_priority(),
+            # 'annotations': self.record.get('annotations', []),
+            'tags': self.get_tags(),
+            'entry': datetime.datetime.fromtimestamp(self.record.get('createdAt'), tz=tzutc()),
+            'due': self.parse_date(self.record.get('duedate')),
+
+            self.AUTHOR: self.record['owner']['uid'],
+            self.BOARD_ID: self.extra['board']['id'],
+            self.BOARD_TITLE: self.extra['board']['title'],
+            self.STACK_ID: self.extra['stack']['id'],
+            self.STACK_TITLE: self.extra['stack']['title'],
+            self.CARD_ID: self.record['id'],
+            self.TITLE: self.record['title'],
+            self.DESCRIPTION: self.record['description'],
+            self.ORDER: self.record['order'],
+            self.ASSIGNEE: ([assignee['participant']['uid'] for assignee in self.record['assignedUsers']] or [None])[0],
```

Comment:
I think it would be more clear to express this as `self.record['assignedUsers'][0]['participant']['uid'] if self.record['assignedUsers'] else None`